dogecoinfoundation/dogecoin.com
# Update main logos & various icons to SVG

## What

This Pull request updates the main logos in the header and footer and various icons to SVG.

## How to test

1. Checkout this branch
2. run `npm install` & `hugo serve -D` in root folder
3. Navigate to the homepage: [http://localhost:1313](http://localhost:1313)
4. Check out:
    - The Dogecoin logo in the header top right
    - The Dogecoin logo in the footer bottom left
    - Right above the footer you'll find the "Find out more about Dogecoin" section, these icons have been updated as well.

Linked issue: #112
